audio-2019.html

```diff
@@ -317,14 +315,10 @@ <h3>
 
         <div id="timeline">
           <h3>Timeline</h3>
-            <p class="todo">Put here a timeline view of all deliverables.</p>
-            <ul class="todo">
-              <li>Month YYYY: First teleconference</li>
-              <li>Month YYYY: First face-to-face meeting</li>
-              <li>Month YYYY: Requirements and Use Cases for FooML</li>
-              <li>Month YYYY: FPWD for FooML</li>
-              <li>Month YYYY: Requirements and Use Cases for BarML</li>
-              <li>Month YYYY: FPWD FooML Primer</li>
+            <ul>
+              <li>Q1 2020: Web Audio API 1.0 completed</li>
+              <li>Q1 2022: Web Audio API 2.0 completed</li>
+              <li>Q1 2020: Audio Device Client completed</li>
```

Comment:
2020 Q1 is probably too agressive.  Q2 or Q3 is probably more realistic.

Reply:
Changed it to Q3 2020